### PR TITLE
feat(Typography): Added type scale

### PR DESCRIPTION
**Pull request by wichniowski**
This PR will add Typography tokens and scale to the design systems docs

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-website/src/pages/foundation/typography/index.mdx [hunk 1]
```diff
@@ -2,4 +2,222 @@
...
 
+import fontBaseTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-base';
+import fontSizeTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-size';
+import fontWeightTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-weight';
+import lineHeightTokens from '../../../../../forma-36-tokens/src/tokens/typography/line-height';
+import fontStackTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-stack';
+import letterSpacingTokens from '../../../../../forma-36-tokens/src/tokens/typography/letter-spacing';
+
+import {
+  Heading,
```
Comment (m10l) [inline packages/forma-36-website/src/pages/foundation/typography/index.mdx]: Heading and subheading aren't used

File: packages/forma-36-website/src/pages/foundation/typography/index.mdx [hunk 2]
```diff
@@ -2,4 +2,222 @@
...
+<Table className="f36-margin-bottom--l">
+  <TableHead>
+    <TableRow>
+      <TableCell>Token</TableCell>
+      <TableCell>CSS Variable</TableCell>
+      <TableCell>Value</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    {Object.keys(fontBaseTokens).map(token => {
```
Comment (m10l) [inline packages/forma-36-website/src/pages/foundation/typography/index.mdx]: Markdown table syntax works and would be preferred but I guess you lose the ability to build up rows dynamically.

File: packages/forma-36-website/src/pages/foundation/typography/index.mdx [hunk 3]
```diff
@@ -2,4 +2,222 @@
 title: 'Typography'
 ---
 
+import fontBaseTokens from '../../../../../forma-36-tokens/src/tokens/typography/font-base';
```
Comment (m10l) [inline packages/forma-36-website/src/pages/foundation/typography/index.mdx]: Can't we just import from the regular package?

File: packages/forma-36-website/src/pages/foundation/typography/index.mdx (hunk 2 above)
Comment (wichniowski) [reply packages/forma-36-website/src/pages/foundation/typography/index.mdx]: yes i'd rather use the table to dynamically build it.. will be less error prone

**Comment by wichniowski** [comment]
@m10l all done

**Pull request merged**